src/components/Bookings.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import "./Bookings.css";
+import Search from "./Search.js";
+import SearchResults from "./SearchResults.js";
+import SearchButton from "./SearchButton.js";
+
+const Bookings = () => {
+  const [bookings, setBookings] = useState(null);
+  useEffect(() => {
+    fetch("https://cyf-react.glitch.me")
+      .then(response => response.json())
+      .then(data => {
+        console.log(data);
```

Comment:
When you are coding in production for a company you should generally remove any `console.log` statements you used fro debugging, nothing wrong with it in here though